README.md

```diff
@@ -19,23 +13,17 @@ This Kubernetes-based version is built using the same approach, and can be
 used as a front-end caching service in a situation where your Kubernetes
 cluster and its ingress controllers have a fast connection to the Internet.
 
-## Usage
```

Comment:
I'd remove `is built using the same approach, and`since there are no references to the machine charm in this doc.

Reply:
I think it would actually be better to keep that and add the reference. So something like "This Kubernetes-based version is built using the same approach as [the machine content-cache charm](https://charmhub.io/content-cache)"

Reply:
The last commit has deleted the reference, but if you get to an agreement and want to keep it with the link I'll do that

Reply:
Yeah, I think it'd be good to include it, thanks.